### PR TITLE
LA-10122 AD based authn for azure mssql

### PR DESCRIPTION
**Pull request by kaif-lb**

### CODE REVIEW

File: sql_user_check_sql_server/database_stats.sh [hunk 1]
```diff
@@ -2,10 +2,12 @@
 
 mode="stats"
 port="1433"
+use_ad_auth=0
```
Comment (arpit-lb) [inline sql_user_check_sql_server/database_stats.sh:5]: We need to explain this flag in README

**Review by arpit-lb — CHANGES_REQUESTED** [review]

File: sql_user_check_sql_server/database_stats.sh (hunk 1 above)
Comment (kaif-lb) [reply sql_user_check_sql_server/database_stats.sh:5]: Provided the explanation that for AD based authentication, -a 1 needs to be set.

File: sql_user_check_sql_server/README.md [hunk 2]
```diff
@@ -7,6 +7,7 @@ Install [sqlcmd](https://learn.microsoft.com/en-us/sql/linux/sql-server-linux-se
 ### Run the script
 
 Below command will ask password.
+For trusting server certificate add -c 1. For Microsoft Entra Authentication add -a 1
```
Comment (arpit-lb) [inline sql_user_check_sql_server/README.md:10]: Mention it as entra authentication or Azure AD authentication

**Review by arpit-lb — APPROVED** [review]

File: sql_user_check_sql_server/README.md (hunk 2 above)
Comment (kaif-lb) [reply sql_user_check_sql_server/README.md]: Done

**Pull request merged**